protocol/avdl/stellar1/bundle.avdl

```diff
@@ -77,4 +78,52 @@ protocol bundle {
     string name;
   }
 
+  // --------------------------------------------------------------------------
+  // AccountBundles
+  // --------------------------------------------------------------------------
+
+  // The same format as in chat1.EncryptedData (and KBFS, Git)
+  record EncryptedAccountBundle {
+    int   v;                           // Version of the CRYPT.
+    bytes e;                           // encrypted msgpacked AccountBundleSecretVersioned (output of secretbox)
+    keybase1.BoxNonce n;               // nonce
+    keybase1.PerUserKeyGeneration gen; // PUK generation that was used
+  }
+
+  // Version of the DATA.
+  enum AccountBundleVersion {
+    V1_1
+  }
+
+  variant AccountBundleSecretVersioned switch (AccountBundleVersion version) {
+    case V1 : AccountBundleSecretV1;
+  }
+
+  record AccountBundleVisibleV1 {
+    BundleRevision revision;
+    Hash prev; // SHA256 of previous msgpack(EncryptedAccountBundle)
+    AccountID accountID;
+    AccountMode mode;
+  }
+
+  record AccountBundleSecretV1 {
+    Hash visibleHash; // SHA256 of msgpack(AccountBundleVisibleV1)
+    AccountID accountID;
+    array<SecretKey> signers;
+    string name;
```

Comment:
I don't this this name field should exist. The name should be stored in 1 place not two. And that place should probably be in BundleSecretEntry.name since you'd want to be able to view and change the name of a mobile-only bundle from a desktop device.